### ISSUE TITLE
Add a new tag (cannot find module providing package github.com/brutella/hc )

### ISSUE THREAD

**Issue by FernFerret** [brutella/hc#129]
**TLDR:** Add a new tag so the "average" user can start using `hc` with go modules.

To install `hc` properly right now with go modules, run: `go get -u github.com/brutella/hc@master`

It's been several years since the 0.1.0 tag was dropped and with the advent of [go modules](https://github.com/golang/go/wiki/Modules); go development has become highly dependent on SemVer tags. I've been led on an interesting chase (which has ultimately given me a lot better understanding of how go modules work) but I ran into this issue with some old code I had (code had not changed, just trying out go modules):

```console
fernferret@computer ~/projects/go/stripper % go install ./stripper
homekit/homekit.go:4:2: unknown import path "github.com/brutella/hc": cannot find module providing package github.com/brutella/hc
```

You can see in your `go.mod` file that you'll be given the latest stable if you just run a `go build`, so I was given 0.1.0 of `hc`:

```
require (
...
	github.com/brutella/hc v0.1.0
...
)
```

It took a little while, but I finally realized that the [tree @ v0.1.0](https://github.com/brutella/hc/tree/v0.1.0) does not contain any go code at the `hc` directory root! (I'd started using `hc` after that and had dependencies on `hc.Config` among others.

**Ok cool**

I now have several options:
* Write an issue and wait for it to get tagged (ok, I'm impatient)
* Clone it to my own repo and own it (NOPE)
* Figure out how to install versions from source <--

After deciding on options 1 + 3, I figured I'd include some info here for other people searching for the same thing.

Unfortunately, I didn't find a lot of information relating to pulling non-tagged go software with all the new blog posts or wiki, but I found a lot of information in `go help modules` and `go help module-get` (which is pointed to from `go help get`) which states you can do the following:

> vX.0.0-yyyymmddhhmmss-abcdefabcdef is used when there is no earlier
versioned commit with an appropriate major version before the target commit.
(This was originally the only form, so some older go.mod files use this form
even for commits that do follow tags.)
> 
> vX.Y.Z-pre.0.yyyymmddhhmmss-abcdefabcdef is used when the most
recent versioned commit before the target commit is vX.Y.Z-pre.
> 
> vX.Y.(Z+1)-0.yyyymmddhhmmss-abcdefabcdef is used when the most
recent versioned commit before the target commit is vX.Y.Z.
> 
> Pseudo-versions never need to be typed by hand: the go command will accept
the plain commit hash and translate it into a pseudo-version (or a tagged
version if available) automatically. This conversion is an example of a
module query.

Cool, the manual states I don't have to go get the hashes by hand... but doesn't tell me how (yet). After some more reading, I got to `go help module-get` which states a bunch of cool stuff:

>For each named package or package pattern, get must decide which version of
the corresponding module to use. By default, get chooses the latest tagged
release version, such as v0.4.5 or v1.2.3. If there are no tagged release
versions, get chooses the latest tagged prerelease version, such as
v0.0.1-pre1. If there are no tagged versions at all, get chooses the latest
known commit.
> 
>This default version selection can be overridden by adding an @version
suffix to the package argument, as in 'go get golang.org/x/text@v0.3.0'.
For modules stored in source control repositories, the version suffix can
also be a commit hash, branch identifier, or other syntax known to the
source control system, as in 'go get golang.org/x/text@master'.
The version suffix @latest explicitly requests the default behavior
described above.

This led me to my final solution:
```console
fernferret@computer ~/projects/go/stripper % go get -u github.com/brutella/hc@master
```

Which yielded the following `go.mod`:
```
require (
...
	github.com/brutella/dnssd v0.0.0-20180519095852-a1eecd10aafc // indirect
	github.com/brutella/hc v0.1.1-0.20180507062808-5b6df487276b
...
)
```

It seems like the modules concept is actually pretty cool and will give tons of flexibility, but I imagine I won't be the first (or last) to discover that a lot of repos (myself included) hadn't been using SemVer tags and hadn't been tagging often.

**Comment by FernFerret** [comment]
Fixed duplicate bit for the correct run command

**Comment by brutella** [comment]
I will release `v1.0.0` of `hc` in the next couple of days/weeks. Sorry that it took so long – I finally have time again to work on it. 🎉

**Comment by theobouwman** [comment]
@brutella when can we expect a new release?

**Comment by brutella** [comment]
I will release the new version of hc once iOS 12.2 ships with the new tv support.

**Comment by brutella** [comment]
Go modules are now supported in [v1.0.0](https://github.com/brutella/hc/releases/tag/v1.0.0).
